src/main/java/mil/nga/tiff/Rasters.java

```diff
@@ -14,6 +14,25 @@
  * @author osbornb
  */
 public class Rasters {
+	/**
+	 * Sample type for pixel
+	 */
+	public enum SampleType {
```

Comment:
Please add an additional format member named e.g. "final int format" that gets initialized to TiffConstants.SAMPLE_FORMAT_FLOAT etc.